### DISCUSSION TITLE
Changing styles on fullscreen?

### DISCUSSION THREAD

**Discussion by oguruma1218** [sachinchoolur/lightGallery#1286] [General]
I'd like to change the styles on full screen mode, but I don't see any classes changing or being added when the user goes to fullscreen.

What I'm trying to do is have a standard inline gallery on page load, but when the user goes to full-screen, I would like to move the captions/subhtml to the right side like in the codepen below.

Is there a method for changing styles/layouts when a user switches to/from fullscreen?

https://codepen.io/sachinchoolur/pen/zYweNJw

**Comment by taskindrp** [comment]
Hi @oguruma1218 , did you solve your issue?

**Comment by sachinchoolur** [comment]
`lg-fullscreen-on` className gets added to the `lg-outer` div when the gallery goes fullscreen. You can use this className to change styles on fullscreen

**Reply by taskindrp** [reply]
is it possible to to zoom out after closing fullscreen in inline gallery? it stays zoomed in event after closing fullscreen. @sachinchoolur
